app/src/main/kotlin/github/com/raccok/dota2androidapp/MainActivity.kt

```diff
@@ -58,7 +69,9 @@ class MainActivity : Activity() {
 
     if (name != null && name.isNotEmpty()) {
```

Comment:
Consider moving your UI into fragments. Instead of straight on an activity.

Reply:
I disagree. Fragments are a pain to deal with when it comes to lifecycles and memory leaks. Only use fragments when you plan on reusing the view + certain business logic.

Reply:
Thanks for your input. Let's keep fragments in mind for later. Also, please recommend using fragments in concrete parts of the code whenever you think it would in fact be beneficial